Makefile

```diff
@@ -11,7 +11,8 @@ TEST_DIR            = test
 CC                  = gcc
 CXX                 = g++
 CFLAGS              =
-CXXFLAGS            =
+#CXXFLAGS	    =
+CXXFLAGS            = -DDEBUG_ITERATION_NUM=20 -DDEBUG_PARTICLE_NUM=0
```

Comment:
I would prefer this to do not be hard coded in the Makefile, but rather that it is passed as an argument to `make`.